src/systems/tracked-controls.js

```diff
@@ -7,6 +7,7 @@ var utils = require('../utils');
  * It maintains a list with the available tracked controllers
  */
 module.exports.System = registerSystem('tracked-controls', {
+  schema: {alwaysAllowIdPrefixes: {type: 'array'}},
```

Comment:
I've been told that `gamepad.pose` should be always available and we can check for `gamepad.pose.hasOrientation` or `gamepad.pose.hasPosition`